lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -334,22 +334,21 @@ def _update_row(attribute_names, attempted_action = 'update')
         end || false
       end
 
-      def destroy(force_delete: false)
+      def destroy(force_delete: false, operated_at: Time.current)
```

Comment:
[nits] For consistency with `#bitemporal_assign_initialize_value`, `current_time:` may be acceptable, but `operated_at:` is also acceptable.

Reply:
I'm wondering if the "current_time" is a good name. Normally, the current time is a unique value and not arbitrary by the caller. Given these circumstances, I thought `operated_at` was appropriate.

Since the `bitemporal_assign_initialize_value` is an internal API, it makes sense to name the argument `current_time`.